### PR TITLE
Update team/about pages.

### PR DESCRIPTION
**Pull request by beshaya**

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -245,7 +245,11 @@
     "contact_us_header": "Contact us",
     "feedback": "If you have any feedback, questions, or would like to help out, please contact us <1>here</1>.",
     "donations_header": "Donations",
-    "donations_details": "We also welcome donations and tips, if you feel so moved. No pressure, truly. We're all volunteers and we don't currently have any funding to work on this project. Thank you in advance!"
+    "donations_details": "We also welcome donations and tips, if you feel so moved. No pressure, truly. Thank you in advance!",
+    "who_are_we_header": "Who are we?",
+    "who_are_we_body": "We are a group of community concious, numerically-minded volunteers. Our project has no funding from any institution - scientific, government, or private. For a list of contributors, see the <1>White Paper</1>.",
```
Comment (jy2wong) [inline src/locales/en.json:250]: concious -> conscious

File: src/locales/en.json [hunk 2]
```diff
@@ -245,7 +245,11 @@
...
+    "affiliations_header": "Disclosure of Affiliations",
+    "affiliations_body": "Our contributors are affiliated or employed by organizations including: One Medical, Massachusetts General Hospital, London School of Hygiene and Tropical Medicine, Google, Matterport, Omada Health, Open Philanthropy Project, Hudson River Traders, Acadamy for Coaching Excellence, and Shapr3D. We do not speak for our affiliates or employers; all statements in this project reflect our own personal views."
```
Comment (jy2wong) [inline src/locales/en.json:252]: Acadamy -> Academy

File: src/locales/en.json (hunk 1 above)
Comment (catherio) [reply src/locales/en.json]: addressed

File: src/locales/en.json (hunk 2 above)
Comment (catherio) [reply src/locales/en.json]: addressed

**Review by catherio — APPROVED** [review]
Love this!! LGTM

**Comment by blanchardjeremy** [comment]
I made a few tweaks. Feel free to revert.

**Comment by blanchardjeremy** [comment]
Also, an idea: What if we linked to the Wired article on the about page and/or the authors section? It's basically the best answer to "how did this project get started" we could ask for.

**Review by catherio — APPROVED** [review]
ty!

**Pull request merged**